File: problemas/ad-hoc/1794/1794.c
#include <stdio.h>

int main(){
    int N, LA, LB, SA, SB;

    scanf("%d %d %d %d %d", &N, &LA, &LB, &SA, &SB);

    if(LA <= N && N <= LB && SA <= N && N <= SB){
        printf("possivel\n");
    }else{
        printf("impossivel\n");
    }

    return 0;
}

File: problemas/ad-hoc/1794/1794.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  List<int> valores =
      stdin.readLineSync()!.split(' ').map((x) => int.parse(x)).toList();
  int la = valores[0];
  int lb = valores[1];
  valores = stdin.readLineSync()!.split(' ').map((x) => int.parse(x)).toList();
  int sa = valores[0];
  int sb = valores[1];

  if (la <= n && n <= lb && sa <= n && n <= sb) {
    print('possivel');
  } else {
    print('impossivel');
  }
}
